resources/definitions/entina_x40.def.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": 2,
+  "name": "ENTINA X40",
+  "inherits": "weedo_base",
+  "metadata":
+  {
+    "visible": true,
+    "author": "ENTINA",
+    "manufacturer": "ENTINA",
+    "file_formats": "text/x-gcode",
+    "machine_extruder_trains": {
+      "0": "weedo_extruder_left_x40",
```

Comment:
These extruders are not available in the database.. Please add them too

Reply:
Thanks, good catch. Fixed now.

Should there be a unit test/check to catch things like undefined extruders and prevent them from slipping through?